src/toplevel_generators.jl

```diff
@@ -158,13 +163,17 @@ function generate_wrapper_load(src_name, pkg_uuid, __source__)
                 end
 
                 # From the available options, choose the best wrapper script
-                select_platform(valid_wrappers)
+                if host_platform !== nothing
+                    select_platform(valid_wrappers, host_platform)
+                else
+                    select_platform(valid_wrappers)
+                end
```

Comment:
```suggestion
                select_platform(valid_wrappers, host_platform)
```

Reply:
I think we did this for performance reasons: https://github.com/JuliaPackaging/JLLWrappers.jl/pull/35#issuecomment-999051946

Reply:
Ah, that's annoying.   Those two things should not be conflated.